x-pack/plugins/licensing/server/license.ts

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { createHash } from 'crypto';
+import { LicenseFeature } from './license_feature';
+import { LICENSE_STATUS, LICENSE_TYPE } from './constants';
+import { LicenseType, ILicense } from './types';
+
+function toLicenseType(minimumLicenseRequired: LICENSE_TYPE | string) {
+  if (typeof minimumLicenseRequired !== 'string') {
+    return minimumLicenseRequired;
+  }
+
+  if (!(minimumLicenseRequired in LICENSE_TYPE)) {
+    throw new Error(`${minimumLicenseRequired} is not a valid license type`);
+  }
+
+  return LICENSE_TYPE[minimumLicenseRequired as LicenseType];
+}
+
+export class License implements ILicense {
+  private readonly hasLicense: boolean;
+  private readonly license: any;
+  private readonly features: any;
```

Comment:
is not `any`, but something with index signature. according to https://github.com/restrry/kibana/blob/ebed1d224a96fd5f91465a5c62a9b266504dc7c7/x-pack/plugins/licensing/server/license.ts#L173

Reply:
Fixed in next PR.